Makefile

```diff
@@ -0,0 +1,33 @@
+DOCKER_REGISTRY?=quay.io
+DOCKER_PROJECT?=jdzon
+
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# This is a requirement for 'setup-envtest.sh' in the test target.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+
+DOCKER_TAG?=v$(shell date +%Y%m%d)-$(shell git rev-parse --short HEAD)
+
+.PHONY: build-dependencies
+build-dependencies:
+	sudo dnf install podman buildah qemu-user-static -y
+
+.PHONY: build-random-server
+build-random-server: build-dependencies 
+	buildah build --jobs=2 --platform=linux/amd64,linux/arm64 --manifest ${DOCKER_REGISTRY}/$(DOCKER_PROJECT)/random-server:$(DOCKER_TAG) poc-demo/edge/random-server
```

Comment:
TBH: I would change os-stats and random-server to a Variable in the start, just in case that some actions on Github actions in the future.

Reply:
@eloycoto - what do you have in mind? To replace separate verbs for os-stats and random-server and have one wildcard verb or just to have variable image names?